test/house_test.rb

```diff
@@ -0,0 +1,93 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:bedroom, 11, '15')
+    @room_3 = Room.new(:living_room, 25, '15')
+    @room_4 = Room.new(:basement, 30, '41')
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_house_has_attributes
+    assert_equal "$400000", @house.price
+    assert_equal "123 sugar lane", @house.address
+    assert_equal ([]), @house.rooms
```

Comment:
you don't need parentheses around arrays, only around hashes (unless you're going to put them around everything)